src/v2/Apps/Artist/Components/ArtistHeader.tsx

```diff
@@ -172,24 +135,16 @@ export const LargeArtistHeader: React.FC<ArtistHeaderProps> = ({
           </Box>
 
           <FollowArtistButton
-            useDeprecatedButtonStyle={false}
             artist={artist}
-            user={user}
-            onOpenAuthModal={() => handleOpenAuth(mediator, artist)}
-          >
-            Follow
-          </FollowArtistButton>
```

Comment:
Do you need to keep this closing tag?

Reply:
No, we were manually adding the "follow" text here, but the `FollowArtistButton` includes that text already -- jsx is self closing.